Project3/index2.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">  
+    <title>3:00 A.M.</title>
+    <div class="card">
+    <h1>3:00 A.M.</h1>
+    <h2>Rise From the Darkness</h2></head>
+<body style="background-color:#000000"; >
+    <p>I just got knocked out by a some dude.<br />
+        He reintroduced himself, "The Name's Life, Everyday Life." <br />
+        But he said, "You can call me anytime."<br />
+        Me: "WUT?"<br />
+        Then he vanished as I opened my eyes. 
+    </p>
+    <img src="Photo2.jpg" alt="AwakeMaybe"><br />
```

Comment:
If you don't include spaces in your `alt` text, screenreaders have a hard time parsing the words.